types/underscore/underscore-tests.ts

```diff
@@ -2219,6 +2223,44 @@ undefinedIdentityIterateeResult; // $ExpectType StringRecord
     extractChainTypes(_.chain(stringRecordList).unique(true, stringRecordPropertyPath)); // $ExpectType ChainType<StringRecord[], StringRecord>
 }
 
+// zip
+{
+    _.zip(simpleStringList, simpleNumberList, stringRecordList); // $ExpectType any[][]
+    _(simpleStringList).zip(simpleNumberList, stringRecordList); // $ExpectType any[][]
+    extractChainTypes(_.chain(simpleStringList).zip(simpleNumberList, stringRecordList)); // $ExpectType ChainType<any[][], any[]>
```

Comment:
Maybe also test the type produced for single-argument calls.

Reply:
Yeah, it would probably be nice (especially in the future when the result might be more exciting than `any[][]`) to validate more than one arity of parameters. Added!